tools/aquainfra_ogc_api_processes/aquainfra_ogc_api_processes.xml

```diff
@@ -0,0 +1,94 @@
+				<option value="peri-conv">peri-conv: Group data to groups based on date</option>
+				<option value="mean-by-group">mean-by-group: Return group average</option>
+				<option value="ts-selection-interpolation">ts-selection-interpolation: Select and Interpolate Time Series</option>
+				<option value="trend-analysis-mk">trend-analysis-mk: Man-Kendall Trend Analysis on Time Series</option>
+				<option value="barplot-trend-results">barplot-trend-results: Visualisation of statistical analysis results</option>
+				<option value="map-trends-static">map-trends-static: Spatial visualisation of regions and data points</option>
+				<option value="map-shapefile-points">map-shapefile-points: Spatial visualisation of regions and data points</option>
+			</param>
+			<when value="points-att-polygon">
+				<param name="regions" label="Study region or study subregions" optional="false" help="URL to the study region, or several regions, to classify your input data into groups of interest. Currently it has to be provided as a shapefile. It can be in any coordinate system and will be transformed to WGS84 during this process." type="data" format="txt"/>
+				<param name="input_data" label="Table to be merged with study region" optional="false" help="URL to the input table containing the in-situ data points with coordinates. Can be provided as Excel file or CSV file (comma-separated text file). The coordinates have to be in WGS84 coordinate system." type="data" format="txt"/>
+				<param name="colname_long" label="Column name for longitude" optional="false" help="Name of the column that contains longitude values (in WGS84)." type="text"/>
+				<param name="colname_lat" label="Column name for latitude" optional="false" help="Name of the column that contains latitude values (in WGS84)." type="text"/>
+			</when>
+			<when value="peri-conv">
+				<param name="input_data" label="Table to be grouped by date, with date colum" help="URL to the table with a column containing a date. It can have other columns which will not be changed during this process." type="data" format="txt"/>
+				<param name="colname_date" label="Date column name" optional="false" help="Column name of the date column in the format defined above. Example: 'visit date'." type="text"/>
+				<param name="date_format" label="Date format" optional="true" help="The date format used to parse the date, i.e. to extract day, month and year from the date column, e.g.  or 'y-m-d' for dates like '1998-08-22' (this is the default) or 'y/m/d' for dates like '1998/08/22'." type="text"/>
+				<param name="group_to_periods" label="Periods to group the data into" optional="true" help="Define the periods that you want the data to be grouped into. Please follow the example: 'Dec-01:Mar-01,Mar-02:May-30,Jun-01:Aug-30,Sep-01:Nov-30' (first three letters of each month, then a minus/hyphen, then the day (two digits), then comma, then the next period)." type="text"/>
```

Comment:
what about a validator here? Maybe even a regex validator?

Reply:
tried but failed :/

Reply:
At leaast, this one did not work: 
`Parameter group_to_periods: Value 'Dec-01:Mar-01,Mar-02:May-30,Jun-01:Aug-30,Sep-01:Nov-30' does not match regular expression ' ^(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)-[0-9][0-9]:(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)-[0-9][0-9](,(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)-[0-9][0-9]:(Jan|Feb|Mar|Apr|May|Jun|Jul|Aug|Sep|Oct|Nov|Dec)-[0-9][0-9])*$ '`

Reply:
Mh, seems to work https://regex101.com/r/TOsg0G/1

Maybe remove the trainling space?

Reply:
That was the problem. I oversaw the spaces.